JohnsonCountyJail.yaml

```diff
@@ -1313,251 +1263,431 @@ associationDefinitions:
-    fqn: "ol.releasedby"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "Rofficerperson"
-    entitySetName: "JohnsonCounty_IowaCity_JCReleasedBy"
+      general.id:
+        type: "general.id"
+        column: "Court"
+    name: "LocatedAt1"
+
+  LocatedAt2:
+    fqn: "ol.locatedat"
+    entitySetName: "JohnsonCounty_IowaCity_JCLocatedAt"
+    src: "JohnsonCounty_IowaCity_JCJailStay"
+    dst: "JohnsonCounty_IowaCity_JCFacility"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "Held_At"
+    name: "LocatedAt2"
+
+  Postedby1:
+    fqn: "ol.postedby"
+    entitySetName: "JohnsonCounty_IowaCity_JCPostedBy"
+    src: "JohnsonCounty_IowaCity_JCBond"
+    dst: "JohnsonCounty_IowaCity_BondPerson"
     propertyDefinitions:
       ol.id:
         type: "ol.id"
-        transforms:
-        - !<transforms.ConcatTransform>
-          columns: ["Rel_Officer_ID", "MNI_No"]
-    name: "JohnsonCounty_IowaCity_JCReleasedby2"
-  JohnsonCounty_IowaCity_JCTransportedby1:
-    fqn: "ol.transportedby"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "TransportingAgency"
-    entitySetName: "JohnsonCounty_IowaCity_JCTransportedby"
+        column: "Person_Post"
+    name: "Postedby1"
+
+  Postedby2:
+    fqn: "ol.postedby"
+    entitySetName: "JohnsonCounty_IowaCity_JCPostedBy"
+    src: "JohnsonCounty_IowaCity_JCBond"
+    dst: "JohnsonCounty_IowaCity_BondBusiness"
     propertyDefinitions:
       ol.id:
         type: "ol.id"
-        transforms:
-        - !<transforms.ConcatTransform>
-          columns: ["Transp_Officer_ID", "Transp_Agency", "Transp_Officer", "MNI_No"]
-    name: "JohnsonCounty_IowaCity_JCTransportedby1"
-  JohnsonCounty_IowaCity_JCTransportedby2:
-    fqn: "ol.transportedby"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "Tofficerperson"
-    entitySetName: "JohnsonCounty_IowaCity_JCTransportedby"
+        column: "Person_Post"
+    name: "Postedby2"
+
+  Postedby3:
+    fqn: "ol.postedby"
+    entitySetName: "JohnsonCounty_IowaCity_JCPostedBy"
+    src: "JohnsonCounty_IowaCity_JCBond"
+    dst: "JohnsonCounty_IowaCity_BondRelative"
     propertyDefinitions:
       ol.id:
         type: "ol.id"
-        transforms:
-        - !<transforms.ConcatTransform>
-          columns: ["Transp_Officer_ID", "Transp_Agency", "Transp_Officer", "MNI_No"]
-    name: "JohnsonCounty_IowaCity_JCTransportedby2"
-
-##############################################
-## AGENCIES/OFFICERS INVOLVED IN JAILTHINGS ##
-##############################################
+        column: "Person_Post"
+    name: "Postedby3"
 
   JohnsonCounty_IowaCity_JCInvolvedIn1:
     fqn: "ol.involvedin"
+    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     src: "ArrestingAgency"
     dst: "JohnsonCounty_IowaCity_JCCharge"
-    entitySetName: "JohnsonCounty_IowaCity_JCInvolvedIn"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
```

Comment:
Can we add "ol.role" and type in as the value, "ArrestingAgency"? Parallel comments for Transporting agency, Releasing agency, Charging agency.